src/main/java/com/danielagapov/spawn/Services/ChatMessage/IChatMessageService.java

```diff
@@ -4,10 +4,14 @@
 import java.util.UUID;
 
 import com.danielagapov.spawn.DTOs.ChatMessageDTO;
+import com.danielagapov.spawn.DTOs.UserDTO;
 
 public interface IChatMessageService {
     public List<ChatMessageDTO> getAllChatMessages();
     public ChatMessageDTO getChatMessageById(UUID id);
     public List<ChatMessageDTO> getChatMessagesByTagId(UUID tagId);
     public ChatMessageDTO saveChatMessage(ChatMessageDTO chatMessage);
+    public void createChatMessageLike(UUID chatMessageId, UUID userId);
+    public List<UserDTO> getChatMessageLikes(UUID chatMessageId);
+    public void deleteChatMessageLike(UUID chatMessageId, UUID userId);
```

Comment:
make this return type `boolean`, as discussed in another comment, to represent whether the deletion was successful

Reply:
Addressed in reply to the other comment.